src/reducer.test.js

```diff
@@ -0,0 +1,75 @@
+import reducer from './reducer';
+
+import { updateTaskTitle, addTask, deleteTask } from './actions';
+
+describe('reducer', () => {
+  describe('updateTaskTitle', () => {
+    it('changes task title', () => {
+      const state = reducer({
+        taskTitle: '',
+      }, updateTaskTitle('New Title'));
+
+      expect(state.taskTitle).toBe('New Title');
+    });
+  });
+
+  describe('addTask', () => {
+    function reduceAddTask(taskTitle) {
+      return reducer({
+        newId: 100,
+        taskTitle,
+        tasks: [],
+      }, addTask());
+    }
+
+    context('with task title', () => {
+      it('appends a new task into tasks', () => {
+        const state = reduceAddTask('New Task');
+
+        expect(state.tasks).toHaveLength(1);
+        expect(state.tasks[0].id).not.toBeUndefined();
+        expect(state.tasks[0].title).toBe('New Task');
+      });
+
+      it('clears task title', () => {
+        const state = reduceAddTask('New task');
+
+        expect(state.taskTitle).toBe('');
+      });
+    });
+
+    context('without task title', () => {
+      it("doesn't work", () => {
+        const state = reduceAddTask('');
+
+        expect(state.tasks).toHaveLength(0);
+      });
+    });
+  });
+
+  describe('deleteTask', () => {
+    context('with existed task ID', () => {
+      it('remove the task frm tasks', () => {
+        const state = reducer({
+          tasks: [
+            { id: 1, title: 'Task' },
+          ],
+        }, deleteTask(1));
+
+        expect(state.tasks).toHaveLength(0);
+      });
+    });
+
+    context('without existed task ID', () => {
+      it("doesn't work", () => {
```

Comment:
```suggestion
      it('doesn't work', () => {
```

섞여있는 것 같네요. Single Queto로 통일하면 좋을 것 같아요